content/docs/user-guide/glossary/workspace.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Workspace
+match: [workspace]
+tooltip: >-
+  Directory containing all your project files e.g. raw datasets, source code, ML
+  models, etc. Typically, it's also a Git repository. It will contain your DVC
+  project. Learn more about the [workspace
+  concept](/doc/user-guide/glossary/workspace) in DVC.
```

Comment:
Note that links should be the same line, I think. At least for regular md content, Prettier enforces that (but it doesn't understand that this is also md so ignores it).

Reply:
> For the links I think it will come down to which items have their own page (basic concepts) and which ones remain as glossary items

Agree, BTW. But don't they all have concepts pages as of now?

Reply:
> it doesn't understand that this is also md so ignores it

Thanks for pointing this out. Tried the shorter version of linking like you suggested with cache here.

> But don't they all have concepts pages as of now?

Technically all the items in the /glossary/ folder have pages and can be linked to. What I meant is some of those items are listed under Basic Concepts in nav and some still only appear as tooltips (and won't have links to the concept page unless we add it).

Reply:
> some of those items are listed under Basic Concepts in nav and some still only appear as tooltips

Oh like "dependency", yeah OK gotcha. So yes, let's link from the tooltip to the page of all the ones we are actually editing in this PR. Thanks!

Reply:
> let's link from the tooltip to the page of all the ones we are actually editing here

OK, I'll use that pattern. Added placeholder links to each for now.